--- bidir.cpp.old	2010-11-27 14:57:11.000000000 +0900
+++ bidir.cpp	2010-11-29 20:59:44.000000000 +0900
@@ -609,7 +609,7 @@
 extern int netagging(vector<Token> & vt);
 
 string
-bidir_postag(const string & s, const vector<ME_Model> & vme, const vector<ME_Model> & chunking_vme, bool dont_tokenize)
+bidir_postag(const string & s, const vector<ME_Model> & vme, const vector<ME_Model> & chunking_vme, bool dont_tokenize, bool no_ner)
 {
   list<string> lt;
   if (dont_tokenize) {
@@ -636,7 +636,9 @@
 
   bidir_chuning_decode_beam(vt, chunking_vme);
   
-  netagging(vt);
+  if (!no_ner) {
+    netagging(vt);
+  }
 
   string tmp;
   for (size_t i = 0; i < vt.size(); i++) {
